scripts/utils.py

```diff
@@ -0,0 +1,40 @@
+import requests, zipfile, sys
+from colorama import Fore, Style
+from tqdm import tqdm
+
+def print_usage():
+    '''
+    Função que printa a chamada correta em caso de o usuário passar o número errado
+    de argumentos
+    '''
+
+    print ('Chamada errada ou ano inválido! \nChamada Correta: ' + Fore.YELLOW +'python3.6 fetch_licitacoes.py <ano>')
+
+def download_zip(url, file_name):
+    '''
+    Função que baixa o arquivo compactado das licitações
+    '''
+    try:
+        chunkSize = 1024
+        r = requests.get(url, stream=True)
```

Comment:
Se for passado um ano que não exista a url, exemplo, 1900 não mostra erro. Algo que tava vendo era solucionar verificando o response, se fosse 404 era de erro e se fosse 200 era que o script tinha conseguido pegar algo

Reply:
verificar com r.status_code poderia resolver esse bug